src/plugins/reporter-rhtsupport.c

```diff
@@ -489,7 +515,8 @@ int main(int argc, char **argv)
         "\n"
         "Option -t uploads FILEs to the already created case on RHTSupport site.\n"
         "The case ID is retrieved from directory specified by -d DIR.\n"
-        "If problem data in DIR was never reported to RHTSupport, upload will fail.\n"
+        "If problem data in DIR was never reported to RHTSupport, you will ask\n"
```

Comment:
`s/you will/you will be asked/`

Reply:
You are mixing conditional with past tense. And even after s/will/would/ will make it future unreal conditional. The correct sentence should be:
``If problem data in DIR is not reported to Red Hat Support, you will be asked to ask to enter case ID to which you want to upload the FILEs.```

Reply:
@xsuchy I really don't think that "you will be asked to ask to enter case ID" is correct. I believe that '``If problem data in DIR is not reported to Red Hat Support, you will be asked to enter case ID...``. (The first is in present simple since it is subordinate clause).